ast/builtins.go

```diff
@@ -1034,6 +1035,18 @@ var Sprintf = &Builtin{
 	),
 }
 
+// Repeat returns the string s, repeated count number of times.
+var Repeat = &Builtin{
+	Name: "repeat",
```

Comment:
Let's make this `strings.repeat`, what do you think?

Reply:
👍 let's use `strings.repeat`. The bare function names are just here for legacy reasons. All new builtins should have a prefix.